templates/zerver/api.html

```diff
@@ -144,6 +151,10 @@ <h1 class="api-page-header">We hear you like APIs...</h1>
   <span class="nx">realm</span><span class="o">:</span> <span class="s1">'{{ external_api_uri_subdomain }}'</span>
 <span class="p">};</span>
 
+<span></span><span class="c1">//remove /api at the end of the realm name.</span>
```

Comment:
Yeah, I'd rather just fix it in the API bindings; seems cleaner than adding and then remembering to remove this.